changelog/7009.misc.md

```diff
@@ -0,0 +1 @@
+Remove duplicate imports from rasa/core/run.py
```

Comment:
Let's remove this changelog file as the change doesn't really affect the user experience.

Reply:
Please merge the latest `master` into your branch as well!

Reply:
Done! @federicotdn :smile: